pkg/cmd/api/spec/api.swagger.json

```diff
@@ -35,19 +35,28 @@
             "name": "app_id",
             "in": "query",
             "required": false,
-            "type": "string"
+            "type": "array",
```

Comment:
It is better to divided auto generate files and manual changed files into two commit.